### ISSUE TITLE
Issues with graphql@15.0.0

### ISSUE THREAD

**Issue by edmundas-ramanauskas** [EasyGraphQL/easygraphql-tester#136]
Our tests started to fail after upgrading to graphql@15.0.0:

```
  ● Test suite failed to run

    Invalid or incomplete introspection result. Ensure that you are passing "data" property of introspection response and no "errors" was returned alongside: undefined

      13 |     },
      14 |   });
    > 15 |   const tester = new EasyGraphQLTester(schema);
```
my guess is that somewhere `introspectionQuery` from `graphql` is used. but in v15 `graphql` doesn't  export `introspectionQuery` anymore, instead it exports `getIntrospectionQuery` function

**Comment by mcshakes** [comment]
Agreed with @edmundas-ramanauskas , I am running into the same error, especially as I try to incorporate this library with mocha/chai. A pity, because it would be perfect for my use case.

**Comment by ash0080** [comment]
use yarn 
and set below in your package.json

"resolutions": {
    "graphql": "^15.4.0"
  },

**Comment by estrada9166** [comment]
A new version has been release with the fix for this! `6.0.0`
